http/src/main/resources/init-resources/crypto-detector-docker-compose.yaml

```diff
@@ -1,7 +1,6 @@
-version: '2'
+version: '2.4'
 services:
   crypto-detector:
     container_name: "${CRYPTO_DETECTOR_SERVER_NAME}"
     image: "${CRYPTO_DETECTOR_DOCKER_IMAGE}"
-    network_mode: host
```

Comment:
Does this need to reference `app_network` somehow?

Reply:
I added `network.yaml` and `network_gce.yaml` because otherwise I'll have to have 2 copies of each docker-compose file